File: pset-1/cash/cash.c
#include <cs50.h>
#include <math.h>
#include <stdio.h>

int main(void)
{
    int cents;
    do
    {
        cents = get_float("Enter your change: ");
    }
    while (cents <= 0);


    int coins = 0;
    while (cents >= 25)
    {
        cents -= 25;
        coins++;
    }
    while (cents >= 10)
    {
        cents -= 10;
        coins++;
    }
    while (cents >= 5)
    {
        cents -= 5;
        coins++;
    }
    while (cents >= 1)
    {
        cents -= 1;
        coins++;
    }

    printf("You will need at least %i coins\n", coins);
}

File: pset-6/cash/cash.py
from cs50 import get_float

while True:
    try:
        input = get_float("Change Owed: ")
        if input > 0:
            break
    except:
        print("", end="")

coins = 0

change = input * 100

while change >= 25:
    change = change - 25
    coins = coins + 1

while change >= 10:
    change = change - 10
    coins = coins + 1

while change >= 5:
    change = change - 5
    coins = coins + 1

while change >= 1:
    change = change - 1
    coins = coins + 1

print(coins)
